test/groovy/MTABuildTest.groovy

```diff
@@ -346,7 +346,7 @@ public class MtaBuildTest extends BasePipelineTest {
         } else if(m.script.contains('MTA_JAR_LOCATION')) {
             return '/env/mta'
         } else if(m.script.contains('which java')) {
-            return '/path/java'
+            return 0
```

Comment:
Just for more context: I guess sh is expected to be called with ```returnStatus: true```. Just a suggestion: in order to make that clear we could check ```m.script.returnStatus``` here and throw an exception if that assertion is violated. Applies also elsewhere below.